### DISCUSSION TITLE
Minified copy of development build of @magicbell/react-headless

### DISCUSSION THREAD

**Discussion by rrgomide** [magicbell/community#136] [Questions] [answered]
I'm getting this `warning` while building my app: `Warning: It looks like you're using a minified copy of the development build of @magicbell/react-headless. When deploying your app to production, make sure to use the production build which is faster and does not print development warnings.`

It's a Remix app using `pnpm workspaces`. I only have it like a dependency in `package.json` like this:
`"@magicbell/react-headless": "^4.3.1",`

How do I solve this?

**Comment by smeijer** [comment, marked as answer]
Hi @rrgomide ,

Please make sure that your `NODE_ENV` is set to `production` when building.
